examples/run_Hahn_echo.py

```diff
@@ -21,16 +24,35 @@
 from SpinCore_pp.ppg import run_spin_echo
 from datetime import datetime
 import numpy as np
-from Instruments import power_control
 from Instruments.XEPR_eth import xepr
-import h5py
 fl = figlist_var()
-p90_us = 4.477
-date = datetime.now().strftime('%y%m%d')
-#{{{these parameters change for each sample
+config = configparser.ConfigParser()
+config.sections()
+config.read('active.ini')
+#{{{importing acquisition parameters
+acq_params = config['acq_params']
+p90_us = float(acq_params['p90_us'])
+deadtime_us = float(acq_params['deadtime_us'])
+adcOffset = float(acq_params['adc_offset'])
+amplitude = float(acq_params['amplitude'])
+deblank_us = float(acq_params['deblank_us'])
+tau_us = float(acq_params['tau_us'])
+repetition_us = float(acq_params['repetition_us'])
+nScans = float(acq_params['nScans'])
+SW_kHz = float(acq_params['sw_khz'])
+acq_ms = float(acq_params['acq_time_ms'])
+nPoints = int(acq_ms*SW_kHz+0.5)
+#{{{create filename and save to config file
 output_name = 'test'
 node_name = 'echo'
```

Comment:
get rid of both of these -- the chemical name should *already* be in the config file.